docs/user/overview/index.md

```diff
@@ -1,52 +1,35 @@
 ---
 id: overview
 title: Overview
-contributors: <Devin Black:devin@k-f.co>
+contributors: <Graham Nelson:graham@k-f.co>
 ---
 
-# User Documentation 
+# User Documentation
+
+Centrifuge V3 enables permissionless, onchain asset management across multiple roles. This section helps users understand how to interact with the protocol depending on who they are and what they want to do. Whether you're issuing a pool, curating assets, investing, or testing the system, this guide will get you started.
```

Comment:
I think permissionless here is a bit unclear, and likewise "across multiple roles"
```suggestion
Centrifuge is the open infrastructure for onchain asset management. This section helps users understand how to interact with the protocol depending on who they are and what they want to do. Whether you're issuing a pool, curating assets, investing, or testing the system, this guide will get you started.
```